components/global/PlacesAndPolygons.vue

```diff
@@ -106,356 +105,266 @@ const getFlagEmoji = (countryCode: string) => {
+            <li>Decimal degree precision to 4 places (≈11m accuracy)</li>
+            <li>Pre-computed shortest distance between point and ocean</li>
+            <li>Multiple name variants where available</li>
+            <li>Unique alphanumeric ID for each point</li>
+          </ul>
 
-    <!-- New section specifically for the interactive features -->
-    <div class="tile is-ancestor is-block-tablet is-flex-desktop mb-0">
-      <div class="tile is-parent border-right is-block-tablet is-flex-desktop">
-        <div class="tile is-child">
-          <div class="horizontal-divider mb-6"></div>
-          <div class="interactive-feature">
-            <div class="interactive-label">Interactive Feature</div>
-            <RandomLocationMap />
+          <div class="is-size-6 mt-3">
+            <table class="table is-bordered mb-0">
+              <caption>
+                Alaska point location CSV file preview with Indigenous names
+                verified by the
+                <a href="https://www.uaf.edu/anlc/"
+                  >Alaska Native Language Center</a
+                >
+              </caption>
+              <thead>
+                <tr>
+                  <th>id</th>
+                  <th>name</th>
+                  <th>alt_name</th>
+                  <th>latitude</th>
+                  <th>longitude</th>
+                  <th>km to ocean</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>AK1</td>
+                  <td>Afognak</td>
+                  <td>Agw'aneq</td>
+                  <td>58.0078</td>
+                  <td>-152.768</td>
+                  <td>1</td>
+                </tr>
+                <tr>
+                  <td>AK2</td>
+                  <td>Akhiok</td>
+                  <td>Kasukuak</td>
+                  <td>56.9455</td>
+                  <td>-154.170</td>
+                  <td>0.8</td>
+                </tr>
+                <tr>
+                  <td>AK3</td>
+                  <td>Akiachak</td>
+                  <td>Akiacuar</td>
+                  <td>60.9094</td>
+                  <td>-161.431</td>
+                  <td>33.5</td>
+                </tr>
+                <tr>
+                  <td>AK4</td>
+                  <td>Akiak</td>
+                  <td>Akiaq</td>
+                  <td>60.9122</td>
+                  <td>-161.214</td>
+                  <td>43</td>
+                </tr>
+                <tr>
+                  <td>AK5</td>
+                  <td>Akutan</td>
+                  <td>Achan-ingiiga</td>
+                  <td>54.1385</td>
+                  <td>-165.778</td>
+                  <td>1.4</td>
+                </tr>
+                <tr>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
+                  <td>&vellip;</td>
```

Comment:
I think this adds a blank row to the bottom of the table, is this intentional?
![image](https://github.com/user-attachments/assets/e14ca02b-38b0-4b4a-9201-ffdeb19426cd)

I'm not sure it is needed - I think the captions make it clear the tables are previews / examples.